amical/data_processing.py

```diff
@@ -197,7 +199,7 @@ def select_data(cube, clip_fact=0.5, clip=False, verbose=True, display=True):
     return cube_cleaned_checked
 
 
-def sky_correction(imA, r1=100, dr=20, verbose=False):
+def sky_correction(imA, r1=100, dr=20):
```

Comment:
you can't just remove an argument from a user-exposed function. If it's effectively useless, then raise an error message when it's used to say it's going away at some point (use `FutureWarning`)

Reply:
Right sorry, for some reason I treated this as a private function (because I've only seen it used inside AMICAL I guess). I suggest to keep it and display the background value when `verbose=True`. I'll commit these changes in a few minutes.